components/sysutils/exfat/patches/use-shared-object.patch

```diff
@@ -0,0 +1,62 @@
+--- tries/dump/Makefile.am.orig	2018-09-15 07:03:24.000000000 +0000
```

Comment:
Can you make a comment here to describe the reason why this patch exists so we can make decisions if we want to keep it or not in the future?

Reply:
Use the code common to driver and utilities from a library defined as a shared object instead of static linking it into each of them.